### ISSUE TITLE
Address field escaped characters not showing

### ISSUE THREAD

**Issue by brjhaverkamp** [nextcloud/contacts#57]
Hello all,

I ran into this little issue with a "missformatted" ADR field.
ADR;TYPE=HOME:;;IJweg 227\, Z;Zwanenburg;;1161 EW;
The "\, Z" part is not visible in the address field in the contact app, so I was not able to remove it.
In the end I had to download, edit and upload again. But I think this is a bug. 
Attached is the complete Vcard.
[010defb5-d436-4195-b7d6-4239766bca9c.vcf.zip](https://github.com/nextcloud/contacts/files/644490/010defb5-d436-4195-b7d6-4239766bca9c.vcf.zip)

Kind regards,

Bert

**Comment by skjnldsv** [comment]
Hey!
You mean the Z in the `IJweg 227, Z` part?

**Comment by brjhaverkamp** [comment]
Indeed, it isn't (or at least wasn't in my case) visible in the contacts 
webview. even after I imported it again from the attached vcard.  So in 
contacts I just saw the "IJweg 227" part. Only when printing the address 
in glabels, I noticed the problem


Is it the same for you?

**Comment by skjnldsv** [comment]
It is displayed correctly for me. :/

**Comment by brjhaverkamp** [comment]
Too bad. Let me try again. I upgraded contacts in the meantime. Maybe it fixed something. It did change how the comma is processed. Ill get back on it.

On 1 January 2017 1:38:27 PM GMT+01:00, "John Molakvoæ" <notifications@github.com> wrote:
>It is displayed correctly for me. :/
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nextcloud/contacts/issues/57#issuecomment-269901568

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by brjhaverkamp** [comment]
OK, it is visible now here too.  thx Case closed.
